R/Rules-class.R

```diff
@@ -2221,14 +2534,16 @@ StoppingList <- function(stop_list, summary) {
 #' @rdname StoppingAll-class
 #'
 #' @param stop_list (`list`)\cr see slot definition.
-#'
+#' @param report_label (`string`) \cr see slot definition.
 #' @export
 #' @example examples/Rules-class-StoppingAll.R
 #'
-StoppingAll <- function(stop_list) {
-  .StoppingAll(stop_list = stop_list)
+StoppingAll <- function(stop_list, report_label = character(0)) {
```

Comment:
why do we have here `character(0)` as default instead of `NA_character_` above?

Reply:
Changed this to NA for StoopingAll and StoppingAny too.